src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -386,7 +414,16 @@ public static PreferenceSettings Load(string filePath)
             settings.CustomPackageFolders = settings.CustomPackageFolders.Distinct().ToList();
 
             return settings;
-		}
+        }
+
+        /// <summary>
+        /// Returns the static Python template file path
```

Comment:
nope, forgot to remove it - I had a `MessageBox.Show()` during testing. Will keep an eye out for these going forward.
It's now removed.